src/util.js

```diff
@@ -0,0 +1,34 @@
+/**
+   Public util functions for the SDK.
+ */
+
+const isObject = val => Object.prototype.toString.call(val) === '[object Object]';
+
+/**
+   Serialize a single attribute in an object query parameter.
+*/
+const serializeAttribute = attribute => {
+  if (isObject(attribute)) {
+    throw new Error('Nested object in query parameter.');
+  } else if (Array.isArray(attribute)) {
+    return attribute.join(',');
+  } else {
+    return attribute;
+  }
+};
+
+/**
+   Serializes an object into a Flex API query parameter value format:
+   key1:val1;key2:val2;key3:array,values,comma,separated.
+*/
+// eslint-disable-next-line import/prefer-default-export
+export const objQueryParam = obj => {
+  if (!isObject(obj)) {
+    throw new Error('Parameter not an object.');
+  }
+
+  return Object.entries(obj)
+    .filter(([, v]) => !!v)
```

Comment:
Good point. Also, I had no idea 0 is falsy.

Reply:
After some thinking, I think this can even be left out. The API should and does handle empty/falsy values and dropping attributes can make debugging dynamic query parameter code harder.